martin/src/sprites/mod.rs

```diff
@@ -198,18 +195,18 @@ async fn parse_sprite(
 }
 
 pub async fn get_spritesheet(
-    sources: impl Iterator<Item = &SpriteSource>,
+    paths: impl Iterator<Item = &PathBuf>,
```

Comment:
is this relevant to this PR?  I am usually ok to mix in a bunch of unrelated changes into smaller / linter / cleanup PRs, but in case of a the complex PRs that already do a lot of fundamental system changes, it is better to move unrelated stuff to a separate PR